gradle-plugin/src/main/java/io/fairyproject/gradle/FairyExtension.java

```diff
@@ -15,15 +15,7 @@
 @Getter
 public class FairyExtension {
 
-    private static String LATEST = "0.5b1";
-
-    static {
-        try {
-            LATEST = MavenUtil.getLatest("core-platform");
-        } catch (Throwable throwable) {
-            throwable.printStackTrace();
-        }
-    }
+    private static String LATEST = "0.5.2b3";
```

Comment:
Ah, but why? you cannot automatically fetch latest version in this case then

Reply:
I’ll have to check again I most likely included debug (hence why still a draft)